docs/guides/high-availability/introduction.adoc

```diff
@@ -12,6 +12,101 @@ Those setups are intended for a transparent network on a single site.
 The {project_name} high-availability guide goes one step further to describe setups across multiple sites.
 While this setup adds additional complexity, that extra amount of high availability may be needed for some environments.
 
+== When to use a multi-site setup
+
+The multi-site deployment capabilities of {project_name} are targeted at use cases that:
+
+* Are constrained to a single
+<@profile.ifProduct>
+AWS Region.
+</@profile.ifProduct>
+<@profile.ifCommunity>
+AWS Region or an equivalent low-latency setup.
+</@profile.ifCommunity>
+* Permit planned outages for maintenance.
+* Fit within a defined user and request count.
+* Can accept the impact of periodic outages.
```

Comment:
Do you want to merge this with `Permit planned outages for maintenance.`?

Reply:
This is taken from the previously agreed text and will therefore not be changed.